cmd/ttn-lw-cli/commands/root.go

```diff
@@ -59,6 +59,9 @@ var (
 		SilenceUsage:      true,
 		Short:             "The Things Network Command-line Interface",
 		PersistentPreRunE: preRun(checkAuth, refreshToken, requireAuth),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
```

Comment:
@htdvisser Added this otherwise running `ttn-lw-cli --help` returns a non-zero exit status code as well (which makes the CI complain).

Reply:
I'm not sure that's the right solution, because now we would have inconsistent error codes when requesting help on different commands.

How about putting this in `main.go`:

```go
help := commands.Root.HelpFunc()
commands.Root.SetHelpFunc(func(cmd *cobra.Command, a []string) {
	help(cmd, a)
	os.Exit(0)
})
```

Reply:
This will not work as expected, since the help function will be called for invalid commands as well.

Reply:
Perhaps something slightly more elaborate, like:
```go
help := commands.Root.HelpFunc()
commands.Root.SetHelpFunc(func(cmd *cobra.Command, a []string) {
	help(cmd, a)
	for _, arg ;= range a {
	    if a == "--help" {
	         os.Exit(0)
	    }
        }
})
```

But this is also getting a bit messy

Reply:
Okay so maybe we should just accept an exit code 2 in CI, or use a different command for testing. Perhaps have CI call `ttn-lw-* config` instead?